nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -116,17 +178,6 @@ define([
          * @param {DOMElement} node - the node that should be used for the left column
          */
         function buildFileTypePanel(node) {
-            const fileTypesDisplay = {},
```

Comment:
Moved to global scope to make the file type display labels available to other functions.